### PR TITLE
NOISSUE - Update UI and Keto database deployment

### PR DESCRIPTION
**Pull request by mteodor**
Add env variable for UI and hook for Postgresql installation to be prior to the Keto.

### CODE REVIEW

File: charts/mainflux/templates/ui-deployment.yaml [hunk 1]
```diff
@@ -20,6 +20,14 @@ spec:
...
             - name: MF_UI_PORT
               value: "{{ .Values.ui.port }}"
+            - name: MF_UI_MQTT_WS_URL
+              value: 'ws://localhost/mqtt'
+            - name: MF_PROXY_AUTH
+              value: "{{ .Values.oidc.enabled }}"
+            - name: MF_UI_APP_PREFIX
+              value: "{{ .Values.ui.prefix }}"
+            - name : MF_PROXY_LOGOUT_URL
+              value: "/oauth2/sign_out?rd=https%3A%2F%2F{{ .Values.oidc.host }}%2Fauth%2Frealms%2F{{ .Values.oidc.realm }}%2Fprotocol%2Fopenid-connect%2Flogout"
```
Comment (manuio) [inline charts/mainflux/templates/ui-deployment.yaml:30]: Can we find a miore readable solution?

File: charts/mainflux/values.yaml [hunk 2]
```diff
@@ -148,6 +148,8 @@ certs:
 ui:
   image: {}
   port: 3000
+  proxy: false
+  prefix: "ui"
```
Comment (manuio) [inline charts/mainflux/values.yaml:152]: There is no logoutUrl?

File: charts/mainflux/values.yaml [hunk 3]
```diff
@@ -407,3 +409,16 @@ postgresqlketo:
...
   persistence:
     size: 1Gi
+  commonAnnotations:
+    helm.sh/hook-weight: "-1"
+    helm.sh/hook: "pre-install, pre-upgrade"
+    helm.sh/hook-delete-policy: "before-hook-creation"
+
+oidc:
+  enabled: false
+  host: mainflux.io
```
Comment (blokovi) [inline charts/mainflux/values.yaml:419]: should we have this host as example?

File: charts/mainflux/values.yaml [hunk 4]
```diff
@@ -407,3 +409,16 @@ postgresqlketo:
       cpu: 25m
   persistence:
     size: 1Gi
+  commonAnnotations:
+    helm.sh/hook-weight: "-1"
+    helm.sh/hook: "pre-install, pre-upgrade"
+    helm.sh/hook-delete-policy: "before-hook-creation"
+
+oidc:
```
Comment (blokovi) [inline charts/mainflux/values.yaml:417]: what is oidc?

File: charts/mainflux/values.yaml (hunk 2 above)
Comment (mteodor) [reply charts/mainflux/values.yaml:152]: because it is constructed with other values

File: charts/mainflux/values.yaml (hunk 3 above)
Comment (mteodor) [reply charts/mainflux/values.yaml:419]: what should I put? example.com?

File: charts/mainflux/values.yaml (hunk 4 above)
Comment (mteodor) [reply charts/mainflux/values.yaml:417]: openid connect

Comment (manuio) [reply charts/mainflux/values.yaml:417]: I'd use `openIdConn`

**Review by manuio — APPROVED** [review]
LGTM

File: charts/mainflux/templates/ui-deployment.yaml [hunk 5]
```diff
@@ -20,6 +20,14 @@ spec:
         - env:
             - name: MF_UI_PORT
               value: "{{ .Values.ui.port }}"
+            - name: MF_UI_MQTT_WS_URL
+              value: 'ws://localhost/mqtt'
```
Comment (blokovi) [inline charts/mainflux/templates/ui-deployment.yaml:24]: should we get this value from values.yaml, like others?

File: charts/mainflux/values.yaml [hunk 6]
```diff
@@ -148,6 +148,9 @@ certs:
 ui:
   image: {}
   port: 3000
+  proxy: false
```
Comment (blokovi) [inline charts/mainflux/values.yaml:151]: can you make this variable name more descriptive, maybe `proxyAuth`

**Review by manuio — APPROVED** [review]
LGTM

**Review by blokovi — APPROVED** [review]

**Pull request merged**